dex/src/main/scala/com/wavesplatform/dex/error/MatcherError.scala

```diff
@@ -328,6 +328,26 @@ case class OrderInvalidPrice(ord: Order, prcSettings: OrderRestrictionsSettings)
                    |step price = ${'step -> prcSettings.stepPrice}"""
     )
 
+case class MarketOrderCancel(id: Order.Id)
+    extends MatcherError(marketOrder, commonEntity, disabled, e"The market order ${'id -> id} cannot be cancelled manually")
+
+case class InvalidMarketOrderPrice(mo: Order)
+    extends MatcherError(
+      marketOrder,
+      price,
+      outOfBound,
+      if (mo.orderType == OrderType.BUY)
+        e"""Price of the buy market order
+         |(${'orderPrice -> mo.price}
+         |${'spendAsset -> AssetPair.assetIdStr(mo.getSpendAssetId)})
```

Comment:
Use `Price(mo.assetPair, mo.price)` here. You can see the usage in `DeviantOrderPrice`.

Reply:
Done